source/educators/references/course_development/course_number_guidelines.rst

```diff
@@ -23,17 +23,23 @@ Course numbers have the following guidelines.
   the way that your course number appears in Studio and the LMS, see
   :ref:`Change the Course Number <Change the Course Number>`.
 
-======================
+**********************
 Example Course Numbers
-======================
+**********************
 
 * CS002x
 * BIO1.1x and BIO1.2x
 * 6.002.1x and 6.002.2x
```

Comment:
Should we really recommend putting "x" at the end of every course number? Maybe something like

```suggestion
* CS101
* BI01.1 and BI01.2
* 6.002.1x and 6.002.2x
```

Reply:
Agree, that's not a naming format that is relevant anymore. Accepting the suggestion.